spring-jdbc/src/main/java/org/springframework/jdbc/core/JdbcTemplate.java

```diff
@@ -1067,9 +1067,8 @@ public int update(String sql, Object... args) throws DataAccessException {
 			logger.debug("Calling stored procedure" + (sql != null ? " [" + sql  + "]" : ""));
 		}
 
-		Connection con = DataSourceUtils.getConnection(getDataSource());
 		CallableStatement cs = null;
-		try {
+		try (Connection con = DataSourceUtils.getConnection(getDataSource())) {
```

Comment:
We're not sure yet we want to do that. And we probably won't. Can you push force an update that doesn't use try/resource please?